galaxy.yml

```diff
@@ -1,10 +1,11 @@
 ---
+version: 1.0.0
```

Comment:
You don't need to set a version, one will be generated on the fly. `build-ansible-collection` will overwrite it with the last tag version (plus a potential -devX suffix).